protobufs/livekit_connectors.proto

```diff
@@ -0,0 +1,138 @@
+// Copyright 2025 LiveKit, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/livekit";
+option csharp_namespace = "LiveKit.Proto";
+option ruby_package = "LiveKit::Proto";
+
+import "livekit_agent_dispatch.proto";
+import "livekit_rtc.proto";
+
+service ConnectorsService {
+  rpc DialWhatsAppCall(DialWhatsAppCallRequest) returns (DialWhatsAppCallResponse);
+
+  rpc DisconnectWhatsAppCall(DisconnectWhatsAppCallRequest) returns (DisconnectWhatsAppCallResponse);
+  
+  rpc ConnectWhatsAppCall(ConnectWhatsAppCallRequest) returns (ConnectWhatsAppCallResponse);
+
+  rpc AcceptWhatsAppCall(AcceptWhatsAppCallRequest) returns (AcceptWhatsAppCallResponse);
+  
+  rpc ListWhatsAppCalls(ListWhatsAppCallsRequest) returns (ListWhatsAppCallsResponse);
+}
+
+message DialWhatsAppCallRequest {
+  // The number of the business that is initiating the call
+  string whatsapp_phone_number_id = 1;
+  // The number of the user that is supossed to receive the call
+  string to_phone_number = 2;
+  // The API key of the business that is initiating the call
+  string whatsapp_api_key = 3;
+  // An arbitrary string you can pass in that is useful for tracking and logging purposes.
+  optional string biz_opaque_callback_data = 4;
+  // What LiveKit room should this participant be connected too
+  string room_name = 5;
+  // Optional identity of the participant in LiveKit room
+  string participant_identity= 6;
+  // Optional name of the participant in LiveKit room
+  string participant_name = 7;
+  // Optional user-defined metadata. Will be attached to a created Participant in the room.
+  string participant_metadata = 8;
+  // Optional user-defined attributes. Will be attached to a created Participant in the room.
+  map<string, string> participant_attributes = 9;
+  // Optional agents to dispatch the call to
+  repeated RoomAgentDispatch agents = 10;
+}
+
+message DialWhatsAppCallResponse {
+  // Call ID sent by Meta
+  string whatsapp_call_id = 1;
+}
+
+message DisconnectWhatsAppCallRequest {
+  // Call ID sent by Meta
+  string whatsapp_call_id = 1;
+  string whatsapp_api_key = 2;
+}
+
+message DisconnectWhatsAppCallResponse {
```

Comment:
Why not an emptypb here?

Reply:
No particular reason, I saw both being used in the repo, empty responses as well as emptypb. No strong opinions either, both are fine.